<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 233, 255);</string>
    </property>
    <property name="title">
     <string>Select Location</string>
    </property>
    <widget class="QPushButton" name="searchLocationBut">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Franklin Gothic Medium&quot;;</string>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Franklin Gothic Medium&quot;;</string>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Franklin Gothic Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Zipcode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stateList">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cityList">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="zipList">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 233, 255);</string>
    </property>
    <property name="title">
     <string>Zipcode Statistics</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Franklin Gothic Medium&quot;;</string>
     </property>
     <property name="text">
      <string># of Businesses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Franklin Gothic Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Total Population</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Franklin Gothic Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Average income</string>
     </property>
    </widget>
    <widget class="QListWidget" name="numberBusinesses">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="totalPopulation">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="avIncome">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 233, 255);</string>
    </property>
    <property name="title">
     <string>Filter Categories</string>
    </property>
    <widget class="QPushButton" name="filterCatBut">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply Filter</string>
     </property>
    </widget>
    <widget class="QListWidget" name="categoryList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>701</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 233, 255);</string>
    </property>
    <property name="title">
     <string>Businesses</string>
    </property>
    <widget class="QTableWidget" name="businessList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>681</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(208, 236, 255);</string>
    </property>
    <property name="title">
     <string>Popular Businesses (in Zipcode)</string>
    </property>
    <widget class="QTableWidget" name="popularList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 206, 207);</string>
    </property>
    <property name="title">
     <string>Successful Businesses (in Zipcode)</string>
    </property>
    <widget class="QTableWidget" name="successfulList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
